docs/ui/prepare-migration/dataflow-config.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Configure Dataflow (Optional)
+parent: Prepare Migration Page
+grand_parent: SMT UI
+nav_order: 4
+---
+
+# Setting up connection profiles
+{: .no_toc }
+
+In case of minimal downtime migration, the dataflow jobs launched by the Spanner Migration Tool can be OPTIONALLY configured with custom runtime environment variables such as MaxWorkers, NumWorkers etc. 
+
+{: .highlight }
+The step shows configured (via a green tick) by default, denoting it is optional to configure.
+
+To configure dataflow, first specify the target database in the 'Configure Spanner Database' step. This enables the configure button for the remaining steps. The configurable fields for Dataflow are mentioned below.
+
+![](https://services.google.com/fh/files/misc/dataflow-config.png)
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Host Project Id
+Specify the project id of the VPC that you want to use. This is required in order to use private connectivity. By default, this is assumed to be the same as Spanner project.
```

Comment:
We need to specify this is to be used for shared VPC, and the project used here will be the host project

Reply:
This needs to be specified for all cases: normal as well as shared VPCs. Added a line for shared VPC as well though.